.circleci/config.yml

```diff
@@ -477,6 +481,78 @@ jobs:
           fail_only: true
           failure_message: ":blobonfire: Nightly job failed to release"
 
+  pr-preview:
+    docker:
+      - image: circleci/python:3.8.1
+
+    resource_class: medium+
+
+    working_directory: ~/repo
+
+    steps:
+      - add_ssh_keys:
+          name: Add the read/write Github deploy key
+          fingerprints:
+            - "06:66:05:ea:ba:10:ec:b4:93:06:ab:62:7c:20:33:f0"
+
+      - checkout:
+          name: Checkout Streamlit code
+
+      - update-submodules
+
+      - pre-cache
+      - restore-from-cache
+      - pre-make
+      - make-init
+
+      - run:
+          <<: *make_scss_vars
+
+      - run:
+          name: Create wheel file
+          no_output_timeout: 2h
+          command: |
+            ${SUDO} apt-get install rsync
+            make package
+
+      - run:
+          name: Upload wheel to S3
+          command: |
+            ${SUDO} apt-get install -y awscli
+            aws configure set aws_access_key_id ${CORE_PREVIEWS_S3_KEY_ID}
+            aws configure set aws_secret_access_key ${CORE_PREVIEWS_S3_SECRET_KEY}
+
+            cd lib/dist
+            export WHEELFILE="$(ls -t *.whl | head -n 1)"
+            export PREVIEW_BRANCH="pr-${CIRCLE_PR_NUMBER}"
+            aws s3 cp ${WHEELFILE} s3://core-previews/${PREVIEW_BRANCH}/ --acl public-read
+            cd ../..
+
+            echo -e "https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/${WHEELFILE}" >> S3_URL
+
+      - run:
+          name: Setup preview repo
+          command: |
+            git config --global user.email "austin@streamlit.io"
```

Comment:
We never changed it for nightly so I'm sure it's fine but just pointing out that you'll be the person making all the changes :)

Reply:
Yeah really trying to artificially inflate my Github activity

(I don't think this email matters at all except for aesthetics; we could eventually set up hello@streamlit.io or something, idk)